tailwind.config.js

```diff
@@ -76,5 +76,8 @@ module.exports = {
       },
     },
   },
-  plugins: [require("@tailwindcss/typography")],
+  plugins: [
+    require("tailwind-scrollbar")({ nocompatible: true }),
```

Comment:
why was this added?

Reply:
> why was this added?

Since we are using a custom scrollbar in the select options field.